code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -238,6 +238,10 @@
 	var/ignore_pain = FALSE // when true the rav will not go into crit or take crit damage.
 	var/ignore_pain_state = 0 // how far "dead" the rav has got while ignoring pain.
 
+	// *** Wraith vars ***
+	/// Defines the warp shadow that the Wraith creates with its Place Warp Shadow ability, and teleports to with Hyperposition
+	var/obj/effect/alien/warp_shadow/warp_shadow = null
```

Comment:
"Defines"
Defines what? This holds a reference
Also do
 var/datum/nullrefence
instead of
 var/datum/nullrefence = null

Reply:
Semantics, but sure, easy enough to change.